### PR TITLE
(DOCSP-25733): Update type-safe-query for collections examples

### PR DESCRIPTION
**Pull request by cbush**
### Jira

- https://jira.mongodb.org/browse/DOCSP-25733

### Staged Changes

- https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/tsq-collections/sdk/swift/crud/filter-data/#collections

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [ ] Create Jira ticket for corresponding docs-app-services update(s), if any
- [ ] Checked/updated Admin API
- [ ] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: examples/ios/Examples/QueryEngine.swift [hunk 1]
```diff
@@ -186,17 +189,24 @@ class QueryEngineForTypeSafeQuery: XCTestCase {
         // :snippet-end:
 
         // :snippet-start: tsq-collections-contains
-        let aliOrJamiesTasks = tasks.where {
-            $0.assignee.contains("Ali") || $0.assignee.contains("Jamie")
+        let quickWinTasks = tasks.where {
```
Comment (cbush) [inline examples/ios/Examples/QueryEngine.swift:192]: Note this snippet is supposed to be about collections-contains but was showing string-contains.

File: source/sdk/swift/crud/filter-data.txt [hunk 2]
```diff
@@ -172,14 +174,22 @@ You can search for individual values by element, or search within a range.
    * - ``.contains(_ range:)``
      - Equivalent to the ``BETWEEN`` operator. Evaluates to ``true`` if the property referenced by the expression contains a value that is within the range.
 
+   * - ``.containsAny(in: )``
```
Comment (cbush) [inline source/sdk/swift/crud/filter-data.txt:177]: @dacharyc should we explicitly add `.in()` to this table?

Comment (dacharyc) [reply source/sdk/swift/crud/filter-data.txt:180]: Ack, yes, thank you for pointing this out. That's what I get for moving quickly at the end of the day. Added.

**Review by nlarew — APPROVED** [review]
LGTM

**Pull request merged**